src/XRP/default-xrp-client.ts

```diff
@@ -81,6 +82,10 @@ class DefaultXRPClient implements XRPClientDecorator {
     const request = this.networkClient.GetAccountInfoRequest()
     request.setAccount(account)
 
+    const ledger = new LedgerSpecifier()
```

Comment:
I actually think you could just refactor this to be:
```javascript
const accountData = this.getAccountData(address)
```

And that would simplify this method and reduce the boiler plate where you need to set this.

Reply:
@keefertaylor Not quite sure I follow on this comment. Is that in the scope of the PR?